app/src/main/java/com/tvtracker/api/ImdbInterceptor.kt

```diff
@@ -9,7 +9,7 @@ class ImdbInterceptor: Interceptor {
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
             .newBuilder()
-            .addHeader("x-rapidapi-key", BuildConfig.xrapidapikey)
+//            .addHeader("x-rapidapi-key", BuildConfig.xrapidapikey) // Can't initialize this in BuildConfig class
```

Comment:
This BuildConfig.xrapidapikey was showing an error and I was unable to compile your code which is why I commented it out.

Reply:
Ah, two no-nos here:

1) Do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.
2) Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.